### DISCUSSION TITLE
About the Order of Wannier Projections in non-collinear case

### DISCUSSION THREAD

**Discussion by shahid-sattar** [Chengcheng-Xiao/VASP2WAN90_v2_fix#17] [General]
I just want to clarify that the spinor order in wannier90.wout file using the VASP2WANNIER90 interface will be like the following:
site 1 projection s  (spin_1)
site 1 projection s  (spin_2)
site 1 projection px (spin_1)
site 1 projection px (spin_2)
site 1 projection py (spin_1)
site 1 projection py (spin_2)

Is this different from the standard wannier90 output if I don't use this patch? Like, if not using this patch, the projections will be written as:
site 1 projection s  (spin_1)
site 1 projection px (spin_1)
site 1 projection py (spin_1)
site 1 projection s  (spin_2)
site 1 projection px (spin_2)
site 1 projection py (spin_2)
Kindly confirm this to better understand the outputs. Thanks,

**Comment by Chengcheng-Xiao** [comment]
Yes, the spinor projection (after applied this patch) will be done in the order of:

```
DO site=1, nsites:
    DO ispinor=1, 2:
        CALL cal_proj(...)
```
